kin/client.py

```diff
@@ -95,45 +85,53 @@ def get_config(self):
 
         return status
 
-    def get_account_balances(self, address):
+    def get_minimum_fee(self):
```

Comment:
We return it in stoops since the fee is always specified in stroops. I will add this method to the documentation in the next PR. Many changes here didnt have docs since it was pushed before the blog post